users/api/clients.go

```diff
@@ -124,6 +124,34 @@ func clientsHandler(svc users.Service, r *chi.Mux, logger *slog.Logger, pr *rege
 			api.EncodeResponse,
 			opts...,
 		), "disable_client").ServeHTTP)
+
+		r.Get("/groups/{groupID}", otelhttp.NewHandler(kithttp.NewServer(
+			listMembersByGroupEndpoint(svc),
+			decodeListMembersByGroup,
+			api.EncodeResponse,
+			opts...,
+		), "list_users_by_user_group_id").ServeHTTP)
+
+		r.Get("/channels/{channelID}", otelhttp.NewHandler(kithttp.NewServer(
+			listMembersByChannelEndpoint(svc),
+			decodeListMembersByChannel,
+			api.EncodeResponse,
+			opts...,
+		), "list_users_by_channel_id").ServeHTTP)
+
+		r.Get("/things/{thingID}", otelhttp.NewHandler(kithttp.NewServer(
+			listMembersByThingEndpoint(svc),
+			decodeListMembersByThing,
+			api.EncodeResponse,
+			opts...,
+		), "list_users_by_thing_id").ServeHTTP)
+
+		r.Get("/domains/{domainID}", otelhttp.NewHandler(kithttp.NewServer(
```

Comment:
This endpoint is not documented

Reply:
It is documented here:
https://github.com/WashingtonKK/magistrala/blob/69b579199035c44acc352e3f588f0eba2ddac163/api/openapi/users.yml#L974-L975